components/global_header/left_controls/product_menu/product_menu_list/index.ts

```diff
@@ -45,6 +52,13 @@ function mapStateToProps(state: GlobalState) {
     const showVisitSystemConsoleTour = step === TaskNameMapToSteps[OnboardingTasksName.VISIT_SYSTEM_CONSOLE].STARTED;
     const enableCustomUserGroups = isCustomGroupsEnabled(state);
 
+    const subscription = getCloudSubscription(state);
```

Comment:
Is this call guarded to not execute if not in cloud?

Reply:
This is just the selector, but verified that the subscription call is not present for non cloud.